### ISSUE TITLE
Moving backup export and uploads location to /var/opt/emoncms/backup

### ISSUE THREAD

**Issue by TrystanLea** [emoncms/backup#51]
@borpin I've started the process of implementing this.

First of all, the locations are changed in the master branch of the EmonScripts install scripts
 https://github.com/openenergymonitor/EmonScripts/commit/dc44ca24f387d92d46e4fd8c981a3bb8667758a4

Then the backup module install.sh script creates the folders in the right locations:
 https://github.com/emoncms/backup/blob/master/install.sh

I've tested both import and export here and it works great.

So far this will only work for people building a new system using EmonScripts master branch and master branch of backup module, next step is to work out the update process...

**Comment by borpin** [comment]
Those lines are in the update scripts as well. I've mentioned before the disconnect between the install and update scripts. There need to be a way where the update scripts use the same config as used by the install script.

**Comment by TrystanLea** [comment]
Yes looking at this earlier reminded me of the wish to merge the install and update scripts somehow into a single set of scripts that either install or update as needed...

**Comment by TrystanLea** [comment]
Latest EmonScripts master now uses same config script and backup module config creation has been moved to the backup module install script. This feature is now ready to go live with the next merge to stable.

**Comment by borpin** [comment]
Will you make changes to the existing backup config files on update to move the destination?

Does the install script now create the backup config based on the install config (does that make sense)?

**Comment by TrystanLea** [comment]
Yes the existing backup config file is overwritten. 

Yes the backup/install.sh script creates the backup/config.cfg based on EmonScripts/install/config.ini if it exists. If it does not exist it asks the user to enter relevant locations via command line - in a manual process.

So all automated for EmonScripts users but a manual fall back for custom installations.

**Comment by borpin** [comment]
> Yes the existing backup config file is overwritten.

Worth moving it to a backup (sic) version in case a user has modified it and warning them! 😁
